apple/Sources/FerrostarMapLibreUI/Views/DynamicallyOrientingNavigationView.swift

```diff
@@ -84,7 +94,7 @@ public struct DynamicallyOrientingNavigationView: View, CustomizableNavigatingIn
                     LandscapeNavigationOverlayView(
                         navigationState: navigationState,
                         speedLimit: nil,
-                        showZoom: true,
+                        showZoom: navigationState?.isNavigating == true,
```

Comment:
Personally, I'd actually never like to show the zoom buttons, and rely on pinch/tap to zoom in all cases. 

I wonder if a delegate query would be a better choice 

(or maybe init parameter is more conventional in SwiftUI? I'm still pretty new to SwiftUI and not confident on the conventions)

Reply:
I agree, I'm not a fan of zoom buttons either, but I wanted to keep it as close as possible to the current setup first :)

Reply:
All for that. The only reason I added the zoom buttons in the first place was to include them in the available UI components and to provide a simple verification for zoom behavior while navigating. 

Long term goal is the existing configurable static zoom as well as a fancier dynamic zoom per activity. E.g. zooms out when as route annotation speed increases. We're almost there on the "current annotation" state, so this probably won't be hard shortly.

This can become as fancy as we want as long as various use cases are supported 👍